### PR TITLE
Avoid "NaN" display when rendering USD value, and speed up conversion

### PR DESCRIPTION
**Pull request by lukemelia**

### CODE REVIEW

File: packages/web-client/app/services/token-to-usd.ts [hunk 1]
```diff
@@ -39,8 +39,10 @@ export default class TokenToUsd extends Service {
   usdConverters = new UsdConverters();
   #registeredHelpers: Set<TokenToUsdHelper> = new Set();
 
-  @task({ maxConcurrency: 1, drop: true }) *pollTask(): any {
+  @task({ maxConcurrency: 1, restartable: true }) *pollTask(): any {
+    console.log('pollTask start');
     while (this.shouldPoll) {
+      console.log('pollTask loop');
```
Comment (backspace) [inline packages/web-client/app/services/token-to-usd.ts:45]: forgotten? 😯

**Review by backspace — APPROVED** [review]
👍🏻 apart from the likely-forgotten logging statements

**Review by Aierie — APPROVED** [review]
Thanks for fixing this! In case I'm missing something - my understanding of this fix is that it allowing the task to be restarted allows new instances of the helper that reference tokens not fetched in a previous polling loop to trigger a fetch of the necessary converters.

**Comment by lukemelia** [comment]
> Thanks for fixing this! In case I'm missing something - my understanding of this fix is that it allowing the task to be restarted allows new instances of the helper that reference tokens not fetched in a previous polling loop to trigger a fetch of the necessary converters.

You've hit the nail on the head. 👍  There is also an optimization that avoids restarting the task if the list of the symbolsToUpdate has not changed.

**Pull request merged**